content/posts/blog/2021-01-23_MesagerieInstantanee-Nuesteorbadespreaplicatie.md

```diff
@@ -0,0 +1,55 @@
+
+La XMPP Standards Foundation și împreună cu mulți alții implicați în această tehnologie deschisă, credem că XMPP este o alegere excelentă ca tehnologie de comunicare, nu doar în ceea ce privește confidențialitatea datelor. XMPP există de peste douăzeci de ani și multă experiență practică a fost adunată în tot acest timp. Aceasta include mulți indicatori cheie pentru alegerea tehnologiei și care permite suveranitatea comunicării, deoarece susține:
+
+- Descentralizarea serviciilor de comunicații (federație)
+- Standardizarea și extensibilitatea tehnologiei
+- Interoperabilitate
+- Inovația și utilizarea dezvoltării deschise
+- Confidențialitatea și controlul, de asemenea, prin utilizarea criptării de la un capăt la altul
+
+> XMPP oferă informații bine definite despre cum să fie gestionată comunicarea într-o rețea. Utilizatorul decide ce program de mesagerie i se potrivește sau unde ar trebui stocate datele.
+
+Spre deosebire de situația actuală în care alte persoane vă sugerează să utilizați aplicația lor preferată de mesagerie, în universul XMPP aveți libertatea de a vă alege aplicația de mesagerie dintre multe opțiuni - alegeți-o pe cea care vă place, nu cea aleasă de ceilalți ca moftul zilei pe Internet. Diferența este că, indiferent de aplicația dumneavoastră de mesagerie sau de cea a prietenilor voștrii, aveți în continuare confortul de a vă afla în aceeași rețea cu toate contactele voastre, indiferent de alegerea de astăzi. Cred că aceasta este o soluție reală.
+
+### Nu este vorba despre aplicație, ci despre tehnologie
+
+În decizie luați în considerare tehnologia pe care intenționați să o utilizați, și apoi decideți ce aplicație de mesagerie vi se potrivește dumneavoastră sau mediului vostru. XMPP și comunitatea sa au acumulat o experiență variată, care a fost cu succes
+utilizată într-o mulțime de programe. Credem că aceasta este o soluție bună pentru majoritatea oamenilor și a organizațiilor de pretutindeni și pentru Internet în ansamblu - și este accesibilă pentru noi toți ca să inovăm în viitor.
+
+> XMPP are nu doar o gamă largă de aplicații de mesagerie; există, de asemenea, diverse programe pentru server și instrumentele (bibliotecile) pentru a vă construi singuri infrastructura.
+
+Nu există un singur mod în care aplicațiile de comunicare trebuie să fie proiectate. Luați o decizie durabilă pentru o tehnologie standardizată - în acest scop, și multe altele, vă recomandăm utilizarea XMPP!
+
+Dacă sunteți interesați să faceți o alegere pentru următorul deceniu, permiteți-mi să vă îndrum spre lectură suplimentară:
+
+- [Noțiuni de bază](https://xmpp.org/getting-started/), XMPP Standards Foundation
+- Articol pe blogul XSF: [It’s all about choices and control](https://xmpp.org/2015/01/its-all-about-choices-and-control/) scris in 2015 de către Laura
+- [Modern XMPP](https://docs.modernxmpp.org/), site web care susține dezvoltatorii și noii veniți să înceapă cu XMPP
+- Comentarii din comunitatea XMPP (nu toate în limba engleză):
```

Comment:
Can you change this to "not in Romanian" instead of English? 
(nu toate în limba engleză)

Reply:
Will